src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -3115,7 +3116,7 @@ private boolean isRateLimitResponse(Response response) {
     // Static helper methods
     //////////////////////////////////////////////////////////////////////
 
-    private static <T> T complete(ListenableFuture<T> future) {
+    static <T> T complete(ListenableFuture<T> future) {
```

Comment:
Its not the good idea to change method visibility for tests

Reply:
agree(, but I don't want to change this sdk a lot because it is fork.